### ISSUE TITLE
Stuck on this error.

### ISSUE THREAD

**Issue by hydropix** [nicolai256/Stable-textual-inversion_win#23]
RuntimeError: Expected all tensors to be on the same device, but found at least two devices, cpu and cuda:0! (when checking argument for argument index in method wrapper__index_select)

Any idea ?

**Comment by nicolai256** [comment]
> RuntimeError: Expected all tensors to be on the same device, but found at least two devices, cpu and cuda:0! (when checking argument for argument index in method wrapper__index_select)
> 
> Any idea ?

are you using a gpu ? 
local or colab?

**Comment by hydropix** [comment]
local with a 3090ti

**Comment by nicolai256** [comment]
> local with a 3090ti

hmm weird, this repo is now being used in VoC https://softology.pro/voc.htm
so u could do it in there, will be much easier for you too 
its a local gui with a huge amount of scripts so maybe try that :)

**Comment by ndemar** [comment]
Update :  I had to change the `--gpus 0,` to `--gpus=1` and it started working.  Eating a lot of VRAM now 

--Same error-- and the VoC download is 404 now. Also using a Nvidia 3090ti on a intel platform.

**Comment by hydropix** [comment]
Great, I had seen on Reddit that you have to indicate gpus=1, but I didn't think the "=" was necessary! It works now! Thanks a lot

**Comment by nicolai256** [comment]
yeah the --gpus thing is different for everyone for some it's 0 for some it's 1, no idea why
